apple/RNSVGRenderable.mm

```diff
@@ -257,42 +257,38 @@ - (void)renderTo:(CGContextRef)context rect:(CGRect)rect
 #endif
     }
 #endif // TARGET_OS_OSX
-    NSUInteger iheight = (NSUInteger)height;
-    NSUInteger iwidth = (NSUInteger)width;
-    NSUInteger iscale = (NSUInteger)scale;
-    NSUInteger scaledHeight = iheight * iscale;
-    NSUInteger scaledWidth = iwidth * iscale;
-    NSUInteger npixels = scaledHeight * scaledWidth;
-    CGRect drawBounds = CGRectMake(0, 0, width, height);
-
-    // Allocate pixel buffer and bitmap context for mask
-    NSUInteger bytesPerPixel = 4;
-    NSUInteger bitsPerComponent = 8;
-    NSUInteger bytesPerRow = bytesPerPixel * scaledWidth;
-    CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
+    // TODO: make it the right way, currently this is a hack to get the correct scale for the mask
+    scale = scale * self.matrix.a * self.svgView.viewBoxTransform.a;
+    CGRect scaledRect = CGRectMake(0, 0, rect.size.width * scale, rect.size.height * scale);
+
+    NSUInteger npixels = scaledRect.size.width * scaledRect.size.height;
     UInt32 *pixels = (UInt32 *)calloc(npixels, sizeof(UInt32));
-    CGContextRef bcontext = CGBitmapContextCreate(
+
+    CGColorSpaceRef colorSpace = CGColorSpaceCreateDeviceRGB();
+    CGContextRef offscreenContext = CGBitmapContextCreate(
         pixels,
-        scaledWidth,
-        scaledHeight,
-        bitsPerComponent,
-        bytesPerRow,
+        scaledRect.size.width,
+        scaledRect.size.height,
+        8,
```

Comment:
It would be nice to leave those exact values as variables so they are more descriptive.